index.js

```diff
@@ -0,0 +1,422 @@
+let buttons = [
```

Comment:
As the `buttons` reference never changes (i.e. it always points to the same array) it worth using `const` declaration keyword there instead of `let`. This way code tells that this variable will never change its reference. Furthermore, one couldn't change it by mistake, which is great too 😎